docs/contact.rst

```diff
@@ -1,2 +1,17 @@
 Contact Us
-**********
\ No newline at end of file
+**********
+
+| Our preferred channels of communication are public :)
+
+| Do you need help?
+| Do you have ideas?
+| Do you have something to say?
+| Do you have something you want to show us?
+
+`Discuss basin3d
+<https://github.com/BASIN-3D/basin3d/discussions>`_ on GitHub!
+
+
+Caught a bug?
+*************
+If you're interested in contributing to basin3d, check out out our `contributing guidelines <https://github.com/BASIN-3D/basin3d/blob/main/CONTRIBUTING.md>`_. It will help explain why, what, and how to get started.
```

Comment:
@heycatwonton don't put the contributing file in a separate PR add it to this pr.  And use a relative reference for the link

Reply:
@vchendrix I made them into relative links, are they correct? Is there a place for the base url? where would that be? For the contributing file, I made it directly on github "add a file" and added it to the project board from the PR, but next time I can make the file on PyCharm instead so it can reference an existing issue.

Reply:
Oops I was thinking that this was a markdown in the code repo.  You should not have a relative link.  Sorry about that.